src/server/pfs/server/pfsserver.proto

```diff
@@ -53,7 +53,11 @@ message PutFileURLTask {
   string dst = 1;
   string datum = 2;
   string URL = 3;
-  repeated string paths = 4;
+  reserved 4;
+  string start_path = 5;
```

Comment:
nit, it seems like it would read better if the fields were ordered as: start path, start offset, end path, end offset.